package.json

```diff
@@ -106,6 +106,11 @@
                     "type": "boolean",
                     "default": true,
                     "description": "Enable watching file change events of the project."
+                },
+                "ballerina.enableLanguageServerCrashReporter": {
+                    "type": "boolean",
+                    "default": true,
+                    "markdownDescription": "Enable crash reports to be sent to a Microsoft online telemtry service. Read privacy statement [here](https://privacy.microsoft.com/en-us/privacystatement)."
```

Comment:
I think we can't refer to Microsoft licensing policies as extensions come under their own licenses https://code.visualstudio.com/license
Shall we check on this?

Reply:
Okay, I can remove that part. And wondering whether we can have a common config to enable/disable telemtry for ballerina plugin ?

Reply:
+1 for a common setting for the extension.